### PR TITLE
Bugfix for qterm crash and correcting errno

### PR DESCRIPTION
**Pull request by bremanandjk**
#### Describe Bug or Feature
* Qterm is crashing, the actual issue is when the first server goes down, it kills the Postgres server too. Thus causing the subsequent server termination failure.
* PBS errno was not right.

#### Describe Your Change
* Adding a condition for stopping DB server only single-server mode, allowing to run for multi-server mode, so that other servers could access the Postgres.
* Updating correct PBS errno.

Testing logs with multiple servers:
[Quickfix_qterm_crash.txt](https://github.com/subhasisb/pbspro/files/4534405/Quickfix_qterm_crash.txt)

### CODE REVIEW

**Comment by bremanandjk** [comment]
TH3 Results: 

image

Multi-server integration branch: 
image

**Comment by bremanandjk** [comment]
[Copying the reply from Subhasis to this new PR.]

@subhasisb Can we change the server to not start DB incase of multi-server? And let init script handle starting and stopping DB?

_Originally posted by @hirenvadalia in https://github.com/subhasisb/pbspro/pull/77_



[Subhasis] >> why should we change that? No need to change starting of DB. We only need to change stopping of DB. Also, we dont need to do anything like stop if last instance is down etc. You wont know if you are the last instance anyway

**Comment by bremanandjk** [comment]
@subhasisb @suresh-thelkar @hirenvadalia 
Please share your reviews in this PR. I have split these changes in separate PR for the qterm crash case so that this fix would enable us to work on multi-server init script.

File: src/server/pbsd_main.c [hunk 1]
```diff
@@ -1878,7 +1878,8 @@ main(int argc, char **argv)
 		shutdown_nodes();
 
 	/* if brought up the DB, take it down */
-	stop_db();
+	if (get_max_servers() == 1)
```
Comment (hirenvadalia) [inline src/server/pbsd_main.c:1881]: Move this if condition inside stop_db(), as there places also who calls stop_db().

**Comment by bremanandjk** [comment]
@subhasisb @hirenvadalia Thanks for approval, 
Ready to merge!

**Pull request merged**